xml/System.Net/HttpListener.xml

```diff
@@ -965,15 +965,10 @@ The following code example demonstrates calling the `Close` method:
       <Docs>
         <summary>Gets a value that indicates whether <see cref="T:System.Net.HttpListener" /> can be used with the current operating system.</summary>
         <value>
-          <see langword="true" /> if <see cref="T:System.Net.HttpListener" /> is supported; otherwise, <see langword="false" />.</value>
+          Returns <see langword="true" /> on all supported platforms.</value>
```

Comment:
To be consistent with other property value descriptions, I'd reword it:

```suggestion
          <see langword="true" /> if the current platform is supported; otherwise, <see langword="false" />.</value>
```

Reply:
Although, according to [this](https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Property), it should be worded as follows:

"Gets a value that indicates whether the current platform is supported."

@mairaw Is that boolean property guidance still valid?